client/admin/permissions-reducer.js

```diff
@@ -14,6 +14,7 @@ export const Permissions = new Record({
   banUsers: false,
   manageMaps: false,
   manageMapPools: false,
+  deleteMaps: false,
```

Comment:
I wonder if this should be called like `deleteAllMaps` or `massDeleteMaps` or something, since we might want to allow a larger group of people delete/deactivate an individual map? Maybe doesn't matter, not sure

Reply:
And I guess looking further it sorta seems like the case to me that these should be separate (since deleting a single map would probably just be flagging it as such, rather than actually removing the row/references elsewhere?)